patches/chrome-browser-profiles-profile_manager.h.patch

```diff
@@ -8,5 +8,5 @@ index cc3c0dfb5be1b31a7dfbc252350a19d62f39d1b8..abf2b024fedcb7c8929a711d47d88d1b
    }
 +  BRAVE_PROFILE_MANAGER_H
```

Comment:
I don't think this is a correct place for this line now, it should go right before `private:`.

Reply:
Fixed in 4855be50917ac30097f470baef638e37531ddcf4